tinder_api/api.py

```diff
@@ -29,110 +30,100 @@ def __init__(self, host= 'https://api.gotinder.com', headers={}, api_token=None)
         if api_token is not None:
             self.authenticate(api_token)
 
-
     # AUTHENTICATION METHODS
 
     def authenticate(self, api_token):
-        '''
+        """
         Sets the authenticate header attribute.
-        '''
+        """
         self._headers.update({self._auth_key: api_token})
 
-
     # SMS authentication
 
     def send_otp_code(self, phone_number):
-        '''
+        """
         Request a one time password from Tinder via SMS.
 
         :param phone_number: the phone number to send the otp to.
-        '''
+        """
         return self.api_request('auth', 'smsSend', data={'phone_number': phone_number})
 
-
     def get_refresh_token(self, phone_number, otp_code):
-        '''
+        """
         Gets the refresh token.
 
         :param phone_number: the user's phone number.
 
         :param otp_code: received by the user on the number provided.
-        '''
-        response_body = self.api_request('auth', 'smsValidate', data={'otp_code': otp_code, 'phone_number': phone_number})
+        """
+        response_body = self.api_request('auth', 'smsValidate',
+                                         data={'otp_code': otp_code, 'phone_number': phone_number})
         return response_body.get("data", {}).get("refresh_token")
 
-
     def get_api_token(self, refresh_token):
-        '''
+        """
         Gets the API token from the refresh token. Will also authenticate.
 
         :param refresh_token: Refresh token obtained from :method:`get_refresh_token`.
-        '''
-        data = {'refresh_token': refresh_token }
+        """
+        data = {'refresh_token': refresh_token}
         response_body = self.custom_request(self._API_ENDPOINTS['authApiPath'] + '/sms', http_verb='POST', data=data)
         api_token = response_body.get("data", {}).get("api_token", None)
         self.authenticate(api_token)
         return api_token
 
-
     # FB authentication
 
-    def get_facebook_auth_token(self, facebook_auth_token, facebook_user_id):
-        '''
+    def get_facebook_auth_token(self, facebook_auth_token):
+        """
         Get api_token using Facebook.
 
         :param facebook_auth_token: the Facebook auth token of the user to authenticate.
-
-        :param facebook_user_id: the Facebook ID of the user to authenticate.
-        '''
-        response_body = self.custom_request('/auth', http_verb='POST', data={'facebook_token': facebook_auth_token, 'facebook_id': facebook_user_id})
-        token = response_body['token']
+        """
+        response_body = self.custom_request(self._API_ENDPOINTS['authApiPath'] + '/facebook',
+                                            http_verb='POST',
+                                            data={'token': facebook_auth_token})
+        token = response_body['data']['api_token']
```

Comment:
One main change is here - the auth endpoint was the old one (this is the latest one, used by other Tinder clients). Also, only the facebook token is required now.